app/src/main/java/com/developersbreach/composeactors/domain/use_case/RemoveFavoriteMovieUseCase.kt

```diff
@@ -0,0 +1,13 @@
+package com.developersbreach.composeactors.domain.use_case
```

Comment:
Okay, fair enough. 
In this scenario, since we don't have any use cases right now in the app, for now place the file just directly inside the domain, if not, since this usecase is shared, have a package `common` inside the `domain`.